src/Utils/Fallback.tsx

```diff
@@ -0,0 +1,34 @@
+export class Fallback {
+  private interval?: ReturnType<typeof setTimeout>;
+  private timeout?: ReturnType<typeof setTimeout>;
+
+  constructor(
+    private delayMs: number,
+    private pollingIntervalMs: number,
+    private pollingCb: () => Promise<boolean>
```

Comment:
Please, use `Callback` instead of `Cb` in the variable name

Reply:
resolved